stories/feedback/ProgressBar.stories.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { action } from '@storybook/addon-actions';
 import StoryContainer from '../storyHelpers/StoryContainer';
 
 import { ProgressBar } from '@sparkpost/matchbox';
 
 storiesOf('Feedback|ProgressBar', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{ getStory() }</StoryContainer>
-  ))
-  .add('Default', withInfo()(() => (
-    <ProgressBar completed={54} color='red' />
-  )));
+  .addDecorator(getStory => <StoryContainer>{getStory()}</StoryContainer>)
```

Comment:
In order to get system props working with the tokens the stories need to be wrapped with `ThemeProvider`. Would be nice to refactor this a bit into the new storybook format using `export`s instead, see the Button stories for an example